test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -797,6 +798,27 @@ public void TensorFlowTransformCifarSavedModel()
             }
         }
 
+        [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))] // TensorFlow is 64-bit only
+        public void TensorFlowGettingSchemaMultipleTimes()
+        {
+            var modelLocation = "cifar_saved_model";
+            var mlContext = new MLContext(seed: 1, conc: 1);
+            var model = TensorFlowUtils.LoadTensorFlowModel(mlContext, modelLocation);
+            var t1 = new Thread(() => TensorFlowUtils.GetModelSchema(mlContext, modelLocation));
+            var t2 = new Thread(() => TensorFlowUtils.GetModelSchema(mlContext, modelLocation));
+            var t3 = new Thread(() => TensorFlowUtils.GetModelSchema(mlContext, modelLocation));
+            var t4 = new Thread(() => TensorFlowUtils.GetModelSchema(mlContext, modelLocation));
+            var t5 = new Thread(() => TensorFlowUtils.GetModelSchema(mlContext, modelLocation));
+            t1.Start();
+            t2.Start();
+            t3.Start();
+            t4.Start();
+            t5.Start();
+            var a = model.GetInputSchema();
+            Assert.NotNull(a);
```

Comment:
You may need to join all the threads you started, and then check to make sure they didn't throw any exceptions during the background thread.